index.py

```diff
@@ -20,7 +23,38 @@ def __init__(self, args):
         mqtt.connect(not args.offline)
         self.modules = None
         self.states = {}
-
+    def handle_conn(self,conn,addr):
+        try:
+            while True:
+                conn.sendall(f"enter target game, options are {list(self.states.keys())}:".encode())
```

Comment:
De main client is niet terminal based, dit zou dan ook als een losse json lijst verstuurd moeten worden